test/utils.sh

```diff
@@ -1,13 +1,41 @@
 #!/bin/bash
-set -euo pipefail
+set -xeuo pipefail
 
 # lookup test-env.sh.yml for explanation.
 export KEY_VALUE_NOT_EXIST=" "
 
+wait_for_it() {
```

Comment:
did this actually change or just moved?

Reply:
moved and changed. main issues:
 eval $*  && return 0 instead of  eval $@ > /dev/null && echo 'Success!' && return 0
1. Because i didnt want to return echo with success because it fail some of the tests that waiting for the right input. 
2. eval $* take all the command as string and not as an array

Reply:
why was it moved? makes it a bit harder to review it.

Reply:
> Because i didnt want to return echo with success because it fail some of the tests that waiting for the right input.

can you elaborate? maybe give an example?  I don't follow

Reply:
i had too, because i insert this function under cli and it didn't recognise it

Reply:
I know we already discussed this, but just noting that bash-lib has a good function for this.

Reply:
i saw it but since it was already in use and implement i used this one. kubernetes-conjur-deploy use this function as well do i didnt want to do dramatic changes...